corehq/pillows/domain.py

```diff
@@ -37,13 +37,14 @@ def get_domain_kafka_to_elasticsearch_pillow(pillow_id='KafkaDomainPillow'):
         index_info=DOMAIN_INDEX_INFO,
         doc_prep_fn=transform_domain_for_elasticsearch
     )
+    change_feed = KafkaChangeFeed(topics=[DOMAIN], group_id='domains-to-es')
```

Comment:
pillows for domain and group are getting them from document_types file instead of topics.